liboptv/include/sortgrid.h

```diff
@@ -0,0 +1,39 @@
+/* Utilities for handling sortgrid */
+
+#ifndef SORTGRID_H
+#define SORTGRID_H
+
+#include "calibration.h"
+#include "parameters.h"
+#include "tracking_frame_buf.h"
+#include "imgcoord.h"
+#include "multimed.h"   
+#include "trafo.h"
+#include "vec_utils.h"
+#include <stdio.h>
+
+typedef struct
+{
+  int pnr;
+  vec3d pos;
+} coord_3d;
+
+
+typedef struct
+{
+  double x, y;
+} pixel_pos;
```

Comment:
Likewise, we can take vec2d out of my orientation PR #78 and just use arrays of that. A structure is superfluous.